frontend/src/scenes/feature-flags/featureFlagLogic.ts

```diff
@@ -869,6 +903,22 @@ export const featureFlagLogic = kea<featureFlagLogicType>([
                 return (featureFlag?.features?.length || 0) > 0
             },
         ],
+        canCreateEarlyAccessFeature: [
+            (s) => [s.featureFlag, s.variants],
+            (featureFlag, variants) => {
+                return (
+                    featureFlag &&
+                    featureFlag.filters.aggregation_group_type_index == undefined &&
+                    variants.length === 0
+                )
+            },
+        ],
+        hasSurveys: [
+            (s) => [s.featureFlag],
+            (featureFlag) => {
+                return (featureFlag?.surveys?.length || 0) > 0
```

Comment:
This selector will not update unless the feature flag changes. So if `surveys` are added / removed this selector will not update. Is that intended?

Reply:
oh good point, the flag is loaded only on mount right now, or when you revisit the feature flag, I'll add the appropriate listeners to update surveys on flag on survey creation, unless you think there's a better way to go about this?

Just noticed that if I delete something, the notebook logic stays mounted so it still shows the survey, need a refresh for it to say not found. Don't know if we want to do anything here as well